hack/ci/gce-project.sh

```diff
@@ -96,7 +96,7 @@ function create_vm {
   then
     gcloud compute disks create "$diskname" \
       --project "$GCP_PROJECT" \
-      --image-project ubuntu-os-cloud --image-family ubuntu-2004-lts \
+      --image-project centos-cloud --image-family centos-stream-8 \
```

Comment:
it works again today

as alternative, maybe we can try ubuntu first and if it failed, then let's switch to centos
how about add variable so that we can switch easily ?

Reply:
I think a variable would be a similar effort to update. An advantage of having it hard-coded is also that there's no expectation that anybody is testing the alternative, so it's more obvious you might be expected to fix things.

I'm happy that this is currently working and a potential option for the future.